cli/standalone/binary.rs

```diff
@@ -416,36 +417,25 @@ impl<'a> DenoCompileBinaryWriter<'a> {
     }
 
     let target = compile_flags.resolve_target();
-    let binary_name = format!("denort-{target}.zip");
-
-    let binary_path_suffix =
-      match crate::version::DENO_VERSION_INFO.release_channel {
-        ReleaseChannel::Canary => {
-          format!(
-            "canary/{}/{}",
-            crate::version::DENO_VERSION_INFO.git_hash,
-            binary_name
-          )
-        }
-        _ => {
-          format!("release/v{}/{}", env!("CARGO_PKG_VERSION"), binary_name)
-        }
-      };
 
-    let download_directory = self.deno_dir.dl_folder_path();
-    let binary_path = download_directory.join(&binary_path_suffix);
+    let exe_name = "denort";
+    let archive_name = format!("{exe_name}-{target}.zip");
```

Comment:
Maybe this could be returned from a helper function in `download_deno_binary` module so we don't have this computed all over the place?